uni/lib/controller/local_storage/database/app_database.dart

```diff
@@ -32,12 +41,28 @@ class AppDatabase {
   /// The version of this database.
   final int version;
 
+  /// Getter to determine if the session is persistent.
+  Future<bool> get persistentSession async {
+    _persistentSession ??=
+        await PreferencesController.getPersistentUserInfo() != null;
+    return _persistentSession!;
```

Comment:
Do not forget to cache the result here, otherwise the variable is not useful

Reply:
I'm already using a null-aware operator